### PR TITLE
Dure variable

### PR DESCRIPTION
**Pull request by BastLast**

### CODE REVIEW

**Comment by BastLast** [comment]
[CodeFactor](https://www.codefactor.io/repository/github/draftbot-a-discord-adventure/draftbot) found an issue: Unreachable code.

It's currently on:
[src\commands\admin\testCommands\Map\TravelTestCommand.js:28](https://github.com/DraftBot-A-Discord-Adventure/DraftBot/blob/5f6cbf5d770005c01db739d86851c8ec1ec8d4af/src/commands/admin/testCommands/Map/TravelTestCommand.js#L28)
Commit 5f6cbf5d770005c01db739d86851c8ec1ec8d4af

**Comment by BastLast** [comment]
> [CodeFactor](https://www.codefactor.io/repository/github/draftbot-a-discord-adventure/draftbot) found an issue: Unreachable code.
> 
> It's currently on:
> [src\commands\admin\testCommands\Map\TravelTestCommand.js:28](https://github.com/DraftBot-A-Discord-Adventure/DraftBot/blob/5f6cbf5d770005c01db739d86851c8ec1ec8d4af/src/commands/admin/testCommands/Map/TravelTestCommand.js#L28)
> Commit [5f6cbf5](https://github.com/DraftBot-A-Discord-Adventure/DraftBot/commit/5f6cbf5d770005c01db739d86851c8ec1ec8d4af)

@romain22222  tu pourras t'en occuper à l'ocaz ? ( pas urgent je sais que tu es en vacance no soucis )

**Comment by BastLast** [comment]
Penser à clarifier points gagnés. (TODO)

**Review by romain22222 — CHANGES_REQUESTED** [review]

File: src/core/smallEvents/botFactsSmallEvent.js [hunk 1]
```diff
@@ -12,28 +12,90 @@ const executeSmallEvent = async function(message, language, entity, seEmbed) {
 
 	const base = JsonReader.smallEvents.botFacts.emote + " " + translationIntroSE.intro[randInt(0, translationIntroSE.intro.length)];
 
-	const outReceived = draftbotRandom.pick(Object.keys(translationBF.possiblesInfos));
-
-	// TODO : changer la string quand on aura passé Tools en Module
-	const commandToEnter = "get" + outReceived.slice(0, 1).toUpperCase() + outReceived.slice(1) + "(entity, language)";
-	const resultToPutInEmbed = await eval(commandToEnter);
+	// const outReceived = draftbotRandom.pick(Object.keys(translationBF.possiblesInfos));
+	const outReceived = "nbPlayersOnYourMap";
```
Comment (romain22222) [inline src/core/smallEvents/botFactsSmallEvent.js:16]:
```suggestion
	const outReceived = draftbotRandom.pick(Object.keys(translationBF.possiblesInfos));
```

**Review by romain22222 — APPROVED** [review]

**Pull request merged**